### DISCUSSION TITLE
List of form fields and http headers.

### DISCUSSION THREAD

**Discussion by starwalkn** [gotenberg/gotenberg#1028] [Q&A] [answered]
Hi! Is there a list of all available form fields and http headers for each or at least a specific version somewhere? This would be very helpful in developing the client.

**Comment by gulien** [comment, marked as answer]
Hello @dcaraxes 👋 if the documentation is not enough, consider checking the [PHP client](https://github.com/gotenberg/gotenberg-php).

In particular:

* Headers: https://github.com/gotenberg/gotenberg-php/blob/main/src/MultipartFormDataModule.php, https://github.com/gotenberg/gotenberg-php/blob/main/src/ApiModule.php
* Form fields: https://github.com/gotenberg/gotenberg-php/tree/main/src/Modules

**Reply by starwalkn** [reply]
That's what I need, thanks!
